crates/nu-command/src/commands/ignore.rs

```diff
@@ -0,0 +1,49 @@
+extern crate unicode_segmentation;
+
+use crate::prelude::*;
+use nu_engine::WholeStreamCommand;
+use nu_errors::ShellError;
+use nu_protocol::Signature;
+
+pub struct Ignore;
+
+impl WholeStreamCommand for Ignore {
+    fn name(&self) -> &str {
+        "ignore"
+    }
+
+    fn signature(&self) -> Signature {
+        Signature::build("ignore")
+    }
+
+    fn usage(&self) -> &str {
+        "Ignore the output of the previous command in the pipeline"
+    }
+
+    fn run(&self, args: CommandArgs) -> Result<OutputStream, ShellError> {
+        let _: Vec<_> = args.input.collect();
+
+        Ok(OutputStream::empty())
+    }
+
+    fn examples(&self) -> Vec<Example> {
+        vec![Example {
```

Comment:
I don't understand this example.

Reply:
Lolol I put the sample in the description instead of where it should go. I'll fix it